internal/services/cdn/cdn_frontdoor_firewall_policy_resource.go

```diff
@@ -89,6 +89,16 @@ func resourceCdnFrontDoorFirewallPolicy() *pluginsdk.Resource {
 				ValidateFunc: validation.IntBetween(5, 1440),
 			},
 
+			// NOTE: 'captcha expiration' is always
+			// enabled no matter what and cannot be disabled for Premium_AzureFrontDoor
+			// and is not supported in Standard_AzureFrontDoor...
+			"captcha_cookie_expiration_in_minutes": {
+				Type:         pluginsdk.TypeInt,
+				Optional:     true,
```

Comment:
could we add a `NOTE: O+C` to explain why this needs to be optional + computed?

Reply:
Fair point, I will add a comment explaining why this is an O+C field.